src/api/worker_api.rs

```diff
@@ -151,6 +152,67 @@ impl WorkerApiClient {
             Err(response.error_for_status().unwrap_err())
         }
     }
+
+    /// Sends the completed computation file to the worker host.
```

Comment:
As there is a convention on the file name, we may prefer to use it
```suggestion
    /// Sends the completed computed.json file to the worker host.
```